streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/MemberController.java

```diff
@@ -53,6 +54,12 @@ public RestResponse memberList(RestRequest restRequest, Member member) {
     return RestResponse.success(userList);
   }
 
+  @PostMapping("candidateUsers")
+  public RestResponse candidateUsers(RestRequest restRequest, Long teamId) {
+    IPage<User> userList = memberService.findCandidateUsers(teamId, restRequest);
```

Comment:
I suggest return `List<User>` instead of `IPage<User>`, The fact is that this does not use the paging function

Reply:
> @wolfboys thanks for your review & suggestions. The truth what you described is right. Before reaching a consensus, I want to express the original intention of this design: The `interface implemented with Pagination` is a superset of `interface implemented without Pagination` , In other words, if there are other calls that need paging, the interface can still be reused. It will lead to some redundancy in the current front-end calling.
> 
> Looking forward to the final conclusion from you.

Happy Chinese new year, thanks for your feedback, My point : This method does not seem to be reusable. That is, it seems to be customized for specific requests, not a generic method (find user not in team),with explicit query conditions, what you think?

Reply:
> > @wolfboys thanks for your review & suggestions. The truth what you described is right. Before reaching a consensus, I want to express the original intention of this design: The `interface implemented with Pagination` is a superset of `interface implemented without Pagination` , In other words, if there are other calls that need paging, the interface can still be reused. It will lead to some redundancy in the current front-end calling.
> > Looking forward to the final conclusion from you.
> 
> Happy Chinese new year, thanks for your feedback, My point : This method does not seem to be reusable. That is, it seems to be customized for specific requests, not a generic method (find user not in team),with explicit query conditions, what you think?

Got it & updated it.
Happy Chinese new year~ & Thanks for the clarification.